ci-operator/step-registry/openshift-extended/rosacli-tests/openshift-extended-rosacli-tests-commands.sh

```diff
@@ -0,0 +1,263 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+export HOME=/tmp/home
+export REPORT_HANDLE_PATH="/usr/bin"
+export ENABLE_PRINT_EVENT_STDOUT=true
+
+# rosacli login
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+# prepare for the future usage on the kubeconfig generation of different workflow
+test -n "${KUBECONFIG:-}" && echo "${KUBECONFIG}" || echo "no KUBECONFIG is defined"
+test -f "${KUBECONFIG}" && (ls -l "${KUBECONFIG}" || true) || echo "kubeconfig file does not exist"
+ls -l ${SHARED_DIR}/kubeconfig || echo "no kubeconfig in shared_dir"
+ls -l ${SHARED_DIR}/kubeadmin-password && echo "kubeadmin passwd exists" || echo "no kubeadmin passwd in shared_dir"
+
+which extended-platform-tests
+
+#set env for rosa which are required by hypershift qe team
+if test -f "${CLUSTER_PROFILE_DIR}/ocm-token"
+then
+    ROSA_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token") || true
+    export ROSA_TOKEN
+fi
+
+
+# create execution directory
+mkdir -p /tmp/output
+cd /tmp/output
+
+echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_TEST_START"
+trap 'echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_TEST_END"' EXIT
+
+# Configure aws
+AWSCRED="${CLUSTER_PROFILE_DIR}/.awscred"
+if [[ -f "${AWSCRED}" ]]; then
+  export AWS_SHARED_CREDENTIALS_FILE="${AWSCRED}"
+  export AWS_DEFAULT_REGION="${CLOUD_PROVIDER_REGION}"
+else
+  echo "Did not find compatible cloud provider cluster_profile"
+  exit 1
+fi
+
+# Log in
+if [[ ! -z "${ROSA_TOKEN}" ]]; then
+  echo "Logging into ${ROSA_LOGIN_ENV} with offline token"
+  rosa login --env "${ROSA_LOGIN_ENV}" --token "${ROSA_TOKEN}"
+  if [ $? -ne 0 ]; then
+    echo "Login failed"
+    exit 1
+  fi
+else
+  echo "Cannot login! You need to specify the offline token ROSA_TOKEN!"
+  exit 1
```

Comment:
Duplicate logic with line 25

Reply:
@yasun1 Line 25 if export the ROSA_TOKEN. Here is checking if the the env variable exists.